child_compassion/models/child_compassion.py

```diff
@@ -263,6 +263,15 @@ class CompassionChild(models.Model):
     ]
 
     is_special_needs = fields.Boolean()
+
+    #Multicompany
```

Comment:
Small travis error (code formatting issues)
```suggestion
    # Multicompany
```